components/nodemanager-service/pgdb/project_delete.go

```diff
@@ -0,0 +1,89 @@
+package pgdb
+
+import (
+	"context"
+
+	project_update_lib "github.com/chef/automate/lib/authz"
+	uuid "github.com/chef/automate/lib/uuid4"
+	log "github.com/sirupsen/logrus"
+)
+
+const deleteProjectNodeAssociation = `
+DELETE FROM nodes_projects np
```

Comment:
This is not immediately related to this PR . . . but shouldn't the key in this relational table be the project's `id`, not the `project_id`?